src/lib/staged_ledger/staged_ledger.ml

```diff
@@ -1949,16 +1949,7 @@ let%test_module "test" =
           [%sexp_of:
             Ledger.init_state
             * User_command.With_valid_signature.t list
-            * int option list]
-        ~shrinker:
```

Comment:
I'm assuming this shrinker was causing problems?

Reply:
Yes, the test data the shrinker generated triggered an assertion failure in `iter_cmds_acc`.

I'm not sure a shrinker is the right approach for these tests, in any case. If I understand, a shrinker is invoked when there's a failure with the original test data, so invoking the shrinker is masking something with the original test data.